silhouette/Graphtec.py

```diff
@@ -182,6 +184,14 @@
 SILHOUETTE_CAMEO4_TOOL_PEN = 7
 SILHOUETTE_CAMEO4_TOOL_ERROR = 255
 
+SILHOUETTE_CAMEO5_TOOL_EMPTY = 0
+SILHOUETTE_CAMEO5_TOOL_RATCHETBLADE = 1
+SILHOUETTE_CAMEO5_TOOL_AUTOBLADE = 2
+SILHOUETTE_CAMEO5_TOOL_DEEPCUTBLADE = 3
+SILHOUETTE_CAMEO5_TOOL_KRAFTBLADE = 4
+SILHOUETTE_CAMEO5_TOOL_ROTARYBLADE = 5
+SILHOUETTE_CAMEO5_TOOL_PEN = 7
+SILHOUETTE_CAMEO5_TOOL_ERROR = 255                                
```

Comment:
@JWBlackman69 these are never used, so there is probably no need to introduce it?

Reply:
Ok, sounds good.  I was just duplicating the lines from Cameo 4